src/frontend/src/optimizer/rule/mod.rs

```diff
@@ -247,7 +245,6 @@ macro_rules! for_all_rules {
             , { BatchPushLimitToScanRule }
             , { PullUpCorrelatedPredicateAggRule }
             , { SourceToKafkaScanRule }
-            , { SourceToIcebergScanRule }
```

Comment:
Why is this removed?

Reply:
Because the logic of this rule is written in IcebergMergeOnReadRewriter